### DISCUSSION TITLE
Form submitted with up-target - how to scroll to the target element?

### DISCUSSION THREAD

**Discussion by vladstudio** [unpoly/unpoly#346] [Q&A] [answered]
Hi! I must be missing something obvious (again), but after user has submitted the form with up-target="#search-results", is there a recommended way to scroll viewport to #search-results?

Thanks!

**Comment by vladstudio** [comment]
my only idea is to add 

`onsubmit="up.reveal('#search-results', {top: true})"`

to the form element. It kinda works!

**Comment by adam12** [comment, marked as answer]
Does adding [`up-scroll="target"`](https://unpoly.com/scroll-option) to the `<form>` help at all?

**Reply by vladstudio** [reply]
It did! I also needed to scroll the element to top, so the final code is 

`up-target="#search-results" 
                    up-reveal-top="true"
                    up-scroll="target"`
